supportbot/utils/message_classification.py

```diff
@@ -2,6 +2,8 @@
 def is_in_support_channel(message, support_channel_ids):
     return message['channel'] in support_channel_ids
 
+def is_first_message_in_thread(message):
+    return not 'thread_ts' in message.keys()
```

Comment:
~~I don't think this should be necessary, since we're filtering to only messages without a subtype:~~

```
event={"type": "message", "subtype": None},
```

~~Thread replies should have a [subtype](https://api.slack.com/events/message#subtypes) of `message_replied`.~~

Never-mind, this is apparently a [Slack bug](https://api.slack.com/events/message/message_replied):
> Bug alert! This event is missing the subtype field when dispatched over the Events API. Until it is fixed, examine message events' thread_ts value. When present, it's a reply. To be doubly sure, compare a thread_ts to the top-level ts value, when they differ the latter is a reply to the former.

Can we leave a comment linking to that page and mentioning that: "As of Dec 2022, this is only required due to a bug in the Slack API"

Reply:
Ah, interesting, sure I'll add a comment about not needing this check eventually.